specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -820,7 +930,7 @@ model MetricValue {
   timestamp?: string;
```

Comment:
Is the TimeSeriesElement contain data points for all dimensionValues or there is a categorization / seperation?